### ISSUE TITLE
Consider using a named function as a callback to sortTodos

### ISSUE THREAD

**Issue by missKatiaPunter** [fac-13/PLHH-ToDo#52]
Great work, guys, it was really hard to find something in your code that needs changing. 22 tests! 100% test coverage! Awesome!

One thing - we feel the task was to use JS sort 'signature', i.e use a named function in logic as a parameter to sort function, possibly several named functions as alternative sorts. We would suggest you consider doing that and explore how sort works and how named functions are used as callbacks.

**Comment by helenzhou6** [comment]
Yes and putting the sortby function stuff in the logic.js rather than dom.js
